Black_Mirror.md

```diff
@@ -6,18 +6,17 @@
 3. The Entire History of You
 
```

Comment:
why  numbers are not removed for the first season?

Reply:
Because this is a numbered list.